### DISCUSSION TITLE
Problema para generar una contraseña para la SSH-KEY

### DISCUSSION THREAD

**Discussion by agusscarava** [INGCOM-UNRN-P1/cursada-2023#26] [Preguntas y respuestas] [answered]
Hola buenas ! 
Les consulto por si alguno tuvo este problema y lo pudo resolver.
Estoy siguiendo los pasos de configuración de GIT y al momento de crear la ssh-key, luego de pedirte la ubicación, te pide que generes una contraseña, y dice "enter passphrase (empty for no passphrase):" y ahi se supone que debería escribir una contraseña pero cuando tecleo no se escribe nada.

![ssh-key error](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141739307/5d290b76-8a2e-4417-90f9-4188be14a2b6)

alguno tiene idea ?
Gracias!

**Comment by martinvilu** [comment]
No lo cierres si no esta contestado :-D

**Comment by martinvilu** [comment, marked as answer]
Está funcionando correctamente, no te muestra nada para no revelar información alguna de tu contraseña.

Créeme que lo estás ingresando, cuando termines métele enter que debiera de funcionar.
